fedimint-core/src/config.rs

```diff
@@ -155,6 +156,8 @@ pub struct GlobalClientConfig {
     /// API endpoints for each federation member
     #[serde(deserialize_with = "de_int_key")]
     pub api_endpoints: BTreeMap<PeerId, PeerUrl>,
+    /// Signing session keys for each federation member
+    pub broadcast_public_keys: BTreeMap<PeerId, PublicKey>,
```

Comment:
Likely needs to be [default](https://serde.rs/attr-default.html) if not present since old APIs won't return this field.

Reply:
never mind, appears to have become an `Option<_>` later on